.gitignore

```diff
@@ -38,4 +38,5 @@
 ##########
 # config #
 ##########
-config.ini
\ No newline at end of file
+config.ini
+*.csv
```

Comment:
Are there csv files you need to ignore? If not, this is a bit to broad to be in the .gitignore. We would want to specify a specific directory, filename, or regex rather than all csv files, as these are a typical way to store data and configuration information.

If you remove this from the .gitignore and commit, I will merge this in.

Thanks!

Reply:
Thats what I was thinking too, I just accidentally made it and wanted your thoughts